### ISSUE TITLE
DisableUpdateMSProducts doesn't seem to work on Windows 10 2004

### ISSUE THREAD

**Issue by a-vrma** [Disassembler0/Win10-Initial-Setup-Script#320]
I have `DisableUpdateMSProducts` in my preset and I have run the script and restarted but I am still shown that "Receive updates for other Microsoft Products when you update Windows" is enabled.
![screenshot of Windows Settings](https://user-images.githubusercontent.com/29077900/84585894-606a1300-ade2-11ea-8501-0ed5f744b21c.png)

When I turn it off manually it doesn't save my setting (I'm not sure why).

**Comment by a-vrma** [comment]
Also this is off-topic but Cortana is back in my Start Menu too. I followed the instructions in <https://winaero.com/blog/uninstall-and-remove-cortana-in-windows-10-version-2004/> and got rid of it.

**Comment by JustMyGithub** [comment]
Not sure whether this is related, MoUsoCoreWorker.exe handles windows Updates now instead of wuauclt.exe - Bu I dont know whether this changed from 1909 to 2004 or earlier (I upgraded 1803 to 2004)

**Comment by Disassembler0** [comment]
Lol, alright. I was puzzled by "*I have DisableUpdateMSProducts in my preset*", because that used to be the default. So congrats, @a-vrma for finding yet another Microsoft bug. 1909 had Microsoft updates disabled by default. 2004 has it *still* disabled by default, but the control displays it as enabled. If you simply try to disable it, nothing changes. You need to manually disable, enable, disable, so the Microsoft update gets registered as a service managed by the update service manager and then it stays registered *and* disabled. `DisableUpdateMSProducts` tweak can't be used to fix the graphical bug, as it reverts the setting to the "factory default", which in this case means that it unregisters the update service, so the control bugs again.

So basically there's no need to change anything in the script and Microsoft needs to sort this one on their side.

**Comment by a-vrma** [comment]
Thanks!
